php-notebook/README.md

```diff
@@ -0,0 +1,135 @@
+<!--- (![docker pulls](https://img.shields.io/docker/pulls/jupyter/base-notebook.svg) ![docker stars](https://img.shields.io/docker/stars/jupyter/base-notebook.svg) [![](https://images.microbadger.com/badges/image/jupyter/base-notebook.svg)](https://microbadger.com/images/jupyter/base-notebook "jupyter/base-notebook image metadata") -->
+
+# Jupyter Notebook PHP Stack
+
+PHP stack based on official [Base Jupyter Notebook Stack](https://github.com/jupyter/docker-stacks/tree/master/base-notebook)
+
+## What it Gives You
+
+* Minimally-functional Jupyter Notebook 5.0.x (e.g., no pandoc for document conversion)
```

Comment:
The Dockerfile is based on jupyter/minimal-notebook which does have pandoc. See README here: https://github.com/jupyter/docker-stacks/tree/master/minimal-notebook

Did you mean to base it on base-notebook?